newIDE/app/public/index.html

```diff
@@ -9,6 +9,17 @@
     <title>GDevelop 5</title>
 
     <style>
+      /**
+       * Whenever the currently selected language is written RTL, this
+       * class is added to specific components, at runtime.
+       *
+       * This class is referenced directly by the `Utils/i18n` module.
+       * Any other module shall use this class indirectly, via `i18n.css`.
+       */
+      .is-rtl {
```

Comment:
That feels really dirty. Can't you add that through some css to pack via webpack than by adding it to the index.html?

Reply:
Fine, the entire CSS code moves to a different file, used by webpack during build.

Reply:
Arthuro555 careful about your wording ;)
For a first time contributor, it's quite rude to get "this feels really dirty" as the very first comment. Choose instead to explain why you think there is a better way: "Seems like we're adding code to index.html, while we want to keep as little code as possible inside to have a near instant display of something on the page (the background GDevelop logo, while the rest is possible). Would it be possible [...]".